_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: post
+authors: [Kris_Jordens]
+title: 'Introduction to API Management'
+image: /img/kris_Jordens.jpg
+tags: [API Management, API Gateway, Products, Responsibilities]
+category: API
+comments: true
+---
+
+        
+# Introduction to API Management
+
+
+## Table of contents
+* [Introduction to API Management](#Introduction-to-API-Management)
+    * [Problem description](#Problem-description)
+    * [Investigation](#Investigation)
+        * [API Management or API Gateway?](#API-Management-or-API-Gateway)
+        * [API landscape without an API management tool](#API-landscape-without-an-API-management-tool)
+        * [API landscape with an API management tool](#API-landscape-with-an-API-management-tool)
+        * [Responsibilities of an API Management tool](#Responsibilities-of-an-API-Management-tool)
+        * [Challenges](#Challenges)
+            * [Multiple API gateways configured](#Multiple-API-gateways-configured)
+            * [Clustering gateways](#Clustering-gateways)            
+    * [API management products](#API-management-products) 
+    * [Hands-on](#Hands-on)
+        * [Installation](#Installation)
+        * [Setup](#Setup)
+        * [Automation](#Automation)
+    * [Conclusion](#Conclusion)
+
+
+## Problem description
+
+In our current digital world where everyone is creating APIs for exposing data internally and for external partners, we should look on how we can manage and control them.
+
+Most of us also know that we not only have to build the API, but we need to foresee additional responsibilities with it. Some that are just a necessary evil. Like for example Authorization, Authentication, or certain routing policies, etc.  
+Keeping an overview on all available API's can become a struggle these days.

```

Comment:
The first sentence doesn't seem right (most of us ...)
do you think it may be better that we change it to "Most of us know that in addition to building the API, we also need to provide additional responsibilities."  ?

Reply:
super 👍